packages/core/lib/commands/develop/run.js

```diff
@@ -1,5 +1,6 @@
 const emoji = require("node-emoji");
 const mnemonicInfo = require("../../mnemonics/mnemonic");
+const configureGanacheOptions = require("../../configureGanacheOptions");
```

Comment:
Also, the name of this module - how would you expect it to be expanded in the future? Is the name appropriate for that file if we do expand it? Is there a better name that would be appropriate?

Reply:
If we are going to expand this file for some other networks in future, then the module name could change. How about something like `configureManagedNetworks`? Any suggestions @eggplantzzz @cds-amal ?

Reply:
What other kinds of utilities do you imagine might belong in that file?

Reply:
I am not able to think of anything specific right now to be included in this file. But in my opinion, this file can be used as a common adapter for the `core/lib` package.

Reply:
I'm just trying to think of a name that will describe all the things that will go in here.

Reply:
Does something like `managedNetworkAdapter` work?